rest/src/main/java/io/syndesis/rest/v1/state/Edition.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.syndesis.rest.v1.state;
+
+/**
+ * One configuration edition of settings used to store state on the client.
+ */
+public abstract class Edition {
```

Comment:
Yeah I was trying to find a good name, the idea always was to periodically regenerate the keys, then each new generation would have `tid=tid+1`, so as they roll in time I thought _Edition_ would be a good name for it.